ansible/install_ansible.sh

```diff
@@ -65,9 +66,14 @@ if [ "`which ansible`" != ""  ]; then
 fi
 
 # incorrect version removed, or no ansible found.
+# enable universe repo
+echo Enabling universe repository.
+sudo add-apt-repository universe
 echo Installing ansible ${REQUIRED_ANSIBLE_VER} required for ceph-ansible ${CEPH_ANSIBLE_BRANCH} branch.
 sudo add-apt-repository -y ppa:ansible/ansible-${REQUIRED_ANSIBLE_VER}
+# Let universe and ansible repo update together
 sudo apt-get update
+sudo apt-get install sshpass pv gawk
```

Comment:
Thanks for the PR @CruciformHawk7. Did you have docker/docker-compose installed on the host? Should we add that too? Also, please refer this link for some of the pre-req: https://github.com/sodafoundation/api/wiki/SODA-Projects-Cluster-Installation-through-Ansible

Also, these packages are required too
apt-get update && apt-get install -y git make curl wget libltdl7 libseccomp2 libffi-dev gawk

Reply:
Hello, 
Yes, I had a few issues with docker and docker-compose. I had to install them manually as well. I did not have any other issues apart from failed containers and zombie Apache processes (probably after playbook failed). The final line could be `apt-get install sshpass pv gawk docker docker.io docker-compose`
Other packages were really not necessary.

Everything seems to be working now :)
![image](https://user-images.githubusercontent.com/9052171/82352443-9c949880-9a1b-11ea-9393-085dae252669.png)